docs/deployment/quickstart-deploy-to-azure.md

```diff
@@ -33,7 +33,7 @@ This article describes how to use the **Publish** tool for one-time deployment.
 
     ![Choose publish target](../deployment/media/quickstart-publish-azure-new.png)
 
-1. Select **Azure App Service (Windows) **and **Next**.
+1. Select **Azure App Service (Windows)** and **Next**.
```

Comment:
@ghogen This is one example where it affected how the text is actually rendered.

Reply:
@Youssef1313 OK, thanks. Could you remove the part that enables markdownlint, so we can just accept the other changes?

Reply:
@ghogen I have deleted the part that enables the workflow for the repo.